Add redoc html generation example (#149)
ePages-de/restdocs-api-spec: README.md

We can use [redoc](https://github.com/Rebilly/ReDoc) to generate an HTML API reference from our OpenAPI specification.

> The [redoc-cli](https://www.npmjs.com/package/redoc-cli) can be used to just bundle or bundle & serve this API reference:

Reviewer: ```suggestion
The [redoc-cli](https://www.npmjs.com/package/redoc-cli) can be used to bundle (and serve) this API reference:
```

```
# Install redoc-cli
npm install -g redoc-cli

> # Just bundle into zero-dependency HTML-file

Reviewer: ```suggestion
# Bundle the documentation into a zero-dependency HTML-file
```
